src/js/gutenberg-plugin.js

```diff
@@ -8,11 +8,14 @@ import {
 	Icon,
 	ToggleControl,
 	BaseControl,
+	Modal,
 } from '@wordpress/components';
 import { __, sprintf } from '@wordpress/i18n';
 import { registerPlugin } from '@wordpress/plugins';
 import { useState, useEffect, useRef } from '@wordpress/element';
 import { store as postAudioStore } from './store/register';
+import TaxonomyControls from '../../includes/Classifai/Blocks/recommended-content-block/inspector-controls/taxonomy-controls';
```

Comment:
I think ultimately this recommended content block will probably go away, due to Azure Personalizer being sunset. I'm wondering if maybe we should pull this component out from the `Blocks` directory and into our general `src/js` directory and it can then be pulled into the block where needed?

Reply:
Yes, I agree